llvmlite/tests/test_refprune.py

```diff
@@ -521,6 +521,37 @@ def test_fanout_raise_5(self):
         mod, stats = self.check(self.fanout_raise_5)
         self.assertEqual(stats.fanout_raise, 2)
 
+    # test case 6 is from https://github.com/numba/llvmlite/issues/1023
```

Comment:
There's no IR or even Python source anywhere in https://github.com/numba/llvmlite/issues/1023 - perhaps this link should be pointing somewhere else?

Reply:
Emm, I mean this test case is from the graph in #1023.

Reply:
Is this pattern generated by the reproducer in https://github.com/numba/numba/issues/9213#issue-1911942120? I don't see this patch to have any effect to that.